test/go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/jarcoal/httpmock v1.3.1
 	github.com/linode/linodego v1.30.0
 	github.com/linode/linodego/k8s v0.0.0-00010101000000-000000000000
+	github.com/stretchr/testify v1.9.0
```

Comment:
This was added in the parent/child project branch (which hasn't been merged to main yet)